docs/guides/genomics-platform/requesting-data/about-our-data.md

```diff
@@ -46,25 +46,31 @@ Somatic VCF files available on St. Jude Cloud have been generated when possible
 
-#### Clinical Genomics Sample Variant Validation and Filtering
+#### Sample Variant Validation and Filtering
 
-For clinical genomics samples (Clinical Pilot and G4K study cohorts), the validation_status is the analyst's determination of whether a variant is actually present in the genome based on available evidence, and not just a sequencing artifact. Note that this doesn't imply anything regarding variant significance, pathogenicity, or relation to disease. When considering evidence for a variant from sequence data from a given platform (WGS, WES, RNA-Seq), the analyst first makes a determination whether the variant should be marked as ‘Good’ or ‘Bad’. To make this assessment, the analyst considers 1) presented quality metric assigned to the variant according to a computationally derived in-house metric; and 2) manual inspection of read evidence within sequencing data from the given sequencing platform. A ‘Good’ assignment to a variant on a given sequencing platform indicates, upon consideration of 1) and 2), the analyst has deemed the information to support a true variant call. Conversely a ‘Bad’ assignment is made if the analyst suspects the contrary. Note that regardless of ‘1’, if a concerning number of germline reads are observed to be present upon examination by the analyst, then a ‘Bad’ assignment is made. Upon consideration of ‘Good’/’Bad’ evidence assignments across each of the sequencing platform data, the analyst assigns each variant into one of the following four categories: ‘Valid’ (analyst determined to constitute a valid variant given available sequencing type data), ‘Likely Valid’ (analyst determined to constitute a valid variant however involving discordant or some disagreement among different sequencing platform data), ‘Putative’ (analyst determined to constitute a valid variant however with discordant or weaker evidence where data from sequencing types are not equally informative), or ‘Invalid’ (analyst determined variant not a valid call) according to the following definitions:
+The validation_status field indicates the status of validating the presence and origin (somatic/germline) of the variant using an orthogonal methodology. Note that this doesn't imply anything regarding variant significance, pathogenicity, or relation to disease. Typically, validation is performed either by secondary sequencing using a custom capture panel designed around the called variants (common in research studies) or by cross-validation using simultaneous sequencing by WGS, WES, and/or RNA-Seq (used in clinical genomics).
+ 
+For validation by a secondary method, "valid" indicates the variant was validated by the secondary method, "putative" indicates that the secondary method could neither confirm nor invalidate the variant (e.g. due to lack of read coverage by the secondary method), and "invalid" indicates a negative result from the secondary method.
```

Comment:
I'm still not a fan of the term "secondary method". I think "secondary sequencing" would be just as accurate and a bit more clear
```suggestion
For validation by secondary sequencing, "valid" indicates the variant was validated by secondary sequencing, "putative" indicates that secondary sequencing could neither confirm nor invalidate the variant (e.g. due to lack of read coverage by the secondary sequencing), and "invalid" indicates a negative result from the secondary sequencing.
```

Reply:
I think there is an important distinction here. For our purposes, we are in fact using additional sequencing as a validation method, but it could be done through other means as well. So I think when talking about the general process of validation, we should call it a "method". Then in places where we describe our method, we can clarify that it is sequencing.